lib/timing.js

```diff
@@ -86,13 +84,9 @@ function display(data) {
         }
     });
 
-    const table = rows.map(function(row) {
-        return row.map(function(cell, index) {
-            return ALIGN[index](cell, widths[index]);
-        }).join(" | ");
-    });
+    const table = rows.map(row => row.map((cell, index) => ALIGN[index](cell, widths[index])).join(" | "));
```

Comment:
Given the double callback and the join at the end, I think this could benefit from being made multiline if possible.